lib/models/feed_formula_model.dart

```diff
@@ -1,5 +1,6 @@
 class FeedIngredient {
   final String name;
+  final double weight;
```

Comment:
Adding the weight as a feed ingredient makes me pause. Feed ingredients don't really have a weight. So this feels a little bit hacky.

Don't change this, I'm happy to accept as-is, just curious to see if it turns into technical debt at some point.